dashboard/package.json

```diff
@@ -39,11 +42,14 @@
   },
   "scripts": {
     "build": "react-app-rewired build",
+    "dev": "npm run server & npm start",
     "doc": "jsdoc -c jsdoc.json",
     "eject": "react-scripts eject",
+    "server": "node server",
     "start": "react-app-rewired start",
-    "test": "react-app-rewired test"
+    "test": "react-app-rewired test"     
```

Comment:
You have trailing whitespace on this line.

Prettier would probably take care of that...does that tool do the `package.json` file, or is it only applied to Javascript files?

Also, you can probably configure your IDE or your editor to strip trailing whitespace automatically when it saves your changes.